### PR TITLE
INBA-773 / Change that adds the param for including comments to the survey call,

### PR DESCRIPTION
**Pull request by Perry5**
Returns a new CSV that includes the commentHistory. The most **current** comment will be returned as part of project data. This CSV only handles the older comments. To generate anything other than a blank CSV, you'll have to create some "Comment and Review" stages, at least two.

### CODE REVIEW

File: backend/app/controllers/products.js [hunk 1]
```diff
@@ -481,7 +481,8 @@ module.exports = {
             const fields = [ // List of CSV columns
                 'subject', 'user', 'surveyName', 'stage', 'question', 'questionType', 'response', 'choiceText',
                 'publicationLink', 'publicationTitle', 'publicationAuthor', 'publicationDate', 'commenter',
-                'commentReason', 'comment', 'date'
+                'commentReason', 'comment', 'historicCommenter1', 'historicReason1', 'historicComment1',
+                'historicCommenter2', 'historicReason2', 'historicComment2', 'date'
```
Comment (sirctseb) [inline backend/app/controllers/products.js]: Why are we capping this at 2 previous comments?

Comment (Perry5) [reply backend/app/controllers/products.js]: This was from the suggestion @jmfadeley made on Slack earlier today I believe. 
"That includes the comment history and returns comments. It only goes back two histories ago. Technically I think the max length for commentHistory is four, discluding the current comment."

File: backend/app/controllers/products.js [hunk 2]
```diff
@@ -513,8 +517,24 @@ module.exports = {
...
+                            commentHistoryExportRow.subject = rowUoa.name;
+                            commentHistoryExportRow.stage = rowStage.title;
+                            commentHistoryExportRow.priorCommenter = priorCommenter.firstName + ' ' + priorCommenter.lastName;
+                            commentHistoryExportRow.priorReason = exportData.body[i].commentHistory[j].reason;
+                            commentHistoryExportRow.priorComment = exportData.body[i].commentHistory[j].text;
+                            commentHistoryExportData.push(commentHistoryExportRow)
+                        }
+                    }
                 }
-                formattedExportRow.date = exportData.body[i].date;
```
Comment (Perry5) [inline backend/app/controllers/products.js]: Did we delete this for a reason?

Comment (Perry5) [reply backend/app/controllers/products.js]: I added it back- we need it

**Comment by jmfadeley** [comment]
Ach. I just forgot to readd it.

**Pull request merged**